terraform-iac/cpy/app/.terraform.lock.hcl

```diff
@@ -2,37 +2,39 @@
 # Manual edits may be lost in future updates.
 
 provider "registry.terraform.io/hashicorp/aws" {
-  version     = "3.28.0"
-  constraints = ">= 3.0.0, ~> 3.0"
+  version     = "3.38.0"
```

Comment:
It looks like [they're on `v3.46.0` now](https://github.com/hashicorp/terraform-provider-aws/releases). Why not go all the way there?

Reply:
3.38.0 is the required minimum to use default tags. I guess we can go all the way. I was mainly just covering my bases

Reply:
That's a pretty good reason for going to `v3.38.0`. We can bump to `v3.46.0` later if that's more convenient.